spec/controllers/web/dashboard/users_controller_spec.rb

```diff
@@ -0,0 +1,177 @@
+      end
+    end
+
+    context 'not authorized' do
+      let(:user) { create(:user, :active) }
+      before { get :index }
+
+      it 'redirect to dashboard root' do
+        expect(response).to redirect_to dashboard_root_path
+      end
+    end
+
+    context 'staff' do
+      before { login_user(user) }
+      before { get :index }
+
+      it 'renders template' do
+        expect(response).to render_template :index
+      end
+
+      it 'returns success status' do
+        expect(response.status).to eq 200
+      end
+    end
+  end
+
+  describe 'GET #show' do
+    context 'authorized' do
+      let!(:candidate) { create(:user, :screening_completed) }
+      before { login_user(user) }
+      before { get :show, params: { id: candidate.id } }
+
+      it 'renders template' do
+        expect(response).to render_template :show
+      end
+
+      it 'returns success status' do
+        expect(response.status).to eq 200
+      end
+    end
+
+    context 'not authorized' do
+      let!(:candidate) { create(:user, :active) }
+      before { get :show, params: { id: user.id } }
+
+      it 'redirect to dashboard root' do
+        expect(response).to redirect_to dashboard_root_path
+      end
+    end
+  end
+
+  describe 'PUT #active' do
+    context 'authorized' do
+      let!(:candidate) { create(:user, :disable) }
+      before { login_user(user) }
+
+      it 'calls Activate operation' do
+        expect(Ops::Developer::Activate).to receive(:call).with(user: candidate)
+        put :activate, params: { id: candidate.id }
+      end
+
+      it 'redirect to users list' do
+        put :activate, params: { id: candidate.id }
+        expect(response).to redirect_to dashboard_users_path
+      end
+
+      it 'flash' do
+        put :activate, params: { id: candidate.id }
+        expect(flash[:success]).to be_present
+      end
+    end
+
+    context 'not authorized' do
+      let!(:candidate) { create(:user, :disable) }
+
+      it 'redirect to dashboard root' do
+        put :activate, params: { id: candidate.id }
+        expect(response).to redirect_to dashboard_root_path
+      end
+    end
+  end
+
+  describe 'PUT #disable' do
+    context 'authorized' do
+      let!(:candidate) { create(:user, :active) }
+      before { login_user(user) }
+
+      it 'calls Activate operation' do
+        expect(Ops::Developer::Disable).to receive(:call).with(user: candidate)
+        put :disable, params: { id: candidate.id }
+      end
+
+      it 'redirect to users list' do
+        put :disable, params: { id: candidate.id }
+        expect(response).to redirect_to dashboard_users_path
+      end
+    end
+
+    context 'not authorized' do
+      let!(:candidate) { create(:user, :active) }
+
+      it 'redirect to dashboard root' do
+        put :disable, params: { id: candidate.id }
+        expect(response).to redirect_to dashboard_root_path
+      end
+    end
+  end
+
+  describe 'PUT #add_role' do
+    let(:role) { 'stuff' }
```

Comment:
add a blank line before `let` and `context`

Reply:
fix